compose/material3/material3/api/desktop/material3.api

```diff
@@ -0,0 +1,1370 @@
+public final class androidx/compose/material3/AlertDialogDefaults {
+	public static final field $stable I
+	public static final field INSTANCE Landroidx/compose/material3/AlertDialogDefaults;
+	public final fun getContainerColor (Landroidx/compose/runtime/Composer;I)J
+	public final fun getIconContentColor (Landroidx/compose/runtime/Composer;I)J
+	public final fun getShape (Landroidx/compose/runtime/Composer;I)Landroidx/compose/ui/graphics/Shape;
+	public final fun getTextContentColor (Landroidx/compose/runtime/Composer;I)J
+	public final fun getTitleContentColor (Landroidx/compose/runtime/Composer;I)J
+	public final fun getTonalElevation-D9Ej5fM ()F
+}
+
+public final class androidx/compose/material3/AlertDialog_skikoKt {
+	public static final fun AlertDialog (Lkotlin/jvm/functions/Function0;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/window/DialogProperties;Lkotlin/jvm/functions/Function2;Landroidx/compose/runtime/Composer;II)V
+	public static final fun AlertDialog-Oix01E0 (Lkotlin/jvm/functions/Function0;Lkotlin/jvm/functions/Function2;Landroidx/compose/ui/Modifier;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;Landroidx/compose/ui/graphics/Shape;JJJJFLandroidx/compose/ui/window/DialogProperties;Landroidx/compose/runtime/Composer;III)V
```

Comment:
1. What `-Oix01E0` mean here?
2. It looks like this validator doesn't handle values of default parameters, right?

Reply:
1. It is how Kotlin compiles a fun with value classes (Color, for example):
![image](https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/624235f9-9340-4386-ba4b-a9500867d18b)

2. Kotlin compiles such functions to `Foo$default`:

![image](https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/cc5c081f-a6b6-4739-bbd4-914f94cca822)

Reply:
2. And it seems, yes, we can't track default values changes. But it is a behaviour change, not API change